src/shared/services/form/form-value.service.ts

```diff
@@ -74,4 +74,26 @@ export class FormValueService {
     return cloneForm;
   }
 
+  sanitiseDynamicLists(caseFields: CaseField[], editForm: any): any {
+    // loop thorught dynamic fields in caseFields
```

Comment:
This code is quite complex. I was thinking we should move it into a class called `ListFieldTypeSanitiser`. And `sanitiseDynamicLists` in `form-value.service.ts` will simply call `listFieldTypeSanitiser.sanitiseLists`. When moving the code in that class, you should remove the comments and replace them with private methods with self explanatory names. Rather than having all in a long method

We can do this as part of the tech debt ticket. Will update the description

Reply:
oh ok thanks